charts/osm/templates/prometheus-deployment.yaml

```
@@ -22,9 +22,20 @@ spec:
       {{- if not (.Capabilities.APIVersions.Has "security.openshift.io/v1") }}
       {{- include "restricted.securityContext" . | nindent 6 }}
       {{- end }}
-      nodeSelector:
-        kubernetes.io/arch: amd64
-        kubernetes.io/os: linux
+      affinity: 
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: kubernetes.io/os
+                  operator: In
+                  values:
+                  - linux
+                - key: kubernetes.io/arch
+                  operator: In
+                  values: 
+                  - amd64
+                  - arm64
```

The Prometheus image also supports more architectures:
```suggestion
                  values: 
                  - amd64
                  - arm64
                  - arm
                  - s390x
```
https://hub.docker.com/layers/prom/prometheus/v2.18.1/images/sha256-575eb2a6d4c2fb7b15a55a2788294925ab148da77b21dc1c92d0557693fdf749?context=explore